### ISSUE TITLE
research: continue CN x1.2 with breadth-veto risk control

### ISSUE THREAD

**Issue by liuh886** [liuh886/alpha_engine#947]
## Objective

Continue the **CN x1.2** model family after the three-mechanism challenger failed the frozen 2026H1 drawdown gate. Test exactly one interpretable risk-control mechanism while keeping the 17-factor ranking signal and all execution economics frozen.

This issue is the authority for development only. It does not reopen Issue #896 or authorize promotion from the rejected certification.

## Prior evidence

The frozen 2026H1 certification of `cn_x1_2_alpha158_three_mechanism` passed 11/12 gates:

- 20bps relative excess: challenger 52.50% vs incumbent 42.52%;
- 60bps relative excess: challenger 47.13% vs incumbent 37.25%;
- maximum drawdown: challenger -16.69% vs incumbent -7.93%;
- failed gate: drawdown worsening 8.76pp > 3pp boundary;
- score and portfolio reproduction passed.

The model family remains named `cn_x1_2`. The rejected candidate identity and evidence must remain preserved.

## Frozen signal

Candidate ID: `cn_x1_2_alpha158_breadth_veto`.

Ranking signal is unchanged from the rejected challenger:

- exact CN x1.1 14-factor `cn_balanced_ohlcv`;
- `qlib_alpha158.cntd30`;
- `qlib_alpha158.cord5`;
- `qlib_alpha158.imin30`;
- total 17 canonical factor IDs.

XGBoost remains frozen:

- objective `rank:ndcg`;
- gain bins 5;
- rounds 100;
- max leaves 31;
- max depth 0;
- min child weight 1.0;
- learning rate 0.05;
- subsample 1.0;
- colsample_bytree 1.0;
- reg alpha 0.0;
- reg lambda 1.0;
- seed 42.

## Single permitted mechanism

Replace the current risk-on rule `votes >= 2 of 3` with:

```
risk_on = breadth_vote AND (long_trend_vote OR medium_momentum_vote)
```

This promotes the existing breadth vote to a veto. It introduces no new threshold:

- long MA: 200 sessions;
- medium momentum: 60 sessions;
- breadth MA: 60 sessions;
- breadth threshold: 50%.

When risk-off, retain the existing 100% CSI300 fallback.

## Frozen economic path

Unchanged:

- selected CN130 candidate pool;
- benchmark CSI300 / `000300`;
- T+1 execution;
- 10-session horizon and rebalance;
- risk-on 4 sectors × 1 name, equal weight;
- 20bps base cost;
- 60bps stress cost;
- governed sector classification;
- no universe, factor, model, execution, cost, or mapping changes.

## Evidence windows

Development evidence:

- 2024H1;
- 2024H2;
- 2025H1;
- 2025H2;
- 2026H1, explicitly treated as consumed development evidence.

Untouched certification holdout:

- 2026H2;
- must not be loaded, diagnosed, or sampled during development;
- certification requires a separate explicit authorization after development support.

## Development gates

The challenger is supported for later certification only if all pass:

- beats same-runtime CN x1.1 incumbent at 20bps;
- beats same-runtime CN x1.1 incumbent at 60bps;
- positive compounded relative excess at 20bps;
- positive compounded relative excess at 60bps;
- at least 4 of 5 development windows have positive relative excess;
- worst maximum drawdown >= -25%;
- drawdown worsening versus incumbent <= 3pp;
- 2026H1 drawdown worsening versus incumbent <= 3pp;
- 2026H1 relative excess beats incumbent at both 20bps and 60bps;
- risk-on relative excess > 0;
- risk-off relative result no worse than transaction-cost drag;
- risk-on share remains between 25% and 80%;
- risk-on active hit rate >= 50%;
- mean Rank IC is non-negative and unchanged except for deterministic retraining noise;
- exact score reproduction passes and matches the frozen 17-factor signal;
- exact portfolio/holdings reproduction passes.

## Stop rules

- no second candidate in this experiment;
- no new factor or Alpha158 search;
- no model parameter, seed, calibration, threshold, cost, universe, mapping, or fallback search;
- no use of 2026H2;
- no automatic promotion;
- no trade-ready claim;
- retain all failed evidence;
- any gate failure ends this candidate.

All outputs remain `research_only=true` and `trade_ready=false`.
